documentation/put.sh

```diff
@@ -92,7 +92,8 @@ up=".."
 
 # unused_doc_comments,unused_attribute: caused by false positives from doc(cfg(...)) attributes.
 export RUSTFLAGS="--cfg published_docs -A unused_imports -A dead_code -A unexpected_cfgs -A unused_doc_comments -A unused_attributes"
-# export RUSTDOCFLAGS=...
+# All doc(cfg(...)) attributes are gated behind `published_docs` cfg – therefore it must be supplied as well.
+export RUSTDOCFLAGS="--cfg published_docs"
```

Comment:
Do we then still need to provide `--cfg published_docs` to `RUSTFLAGS`?

Reply:
Yeah, because `RUSTDOCFLAGS` is propagated only to `godot` crate (and we need it for codegen and whatnot: https://github.com/godot-rust/gdext/issues/1163#issuecomment-2883380964